containers/kubernetes/reference-content/version-support-policy.mdx

```diff
@@ -18,13 +18,24 @@ categories:
 With Scaleway Kubernetes you can use the latest version of Kubernetes as soon as it is officially released and certified by the [Cloud Native Computing Foundation](https://www.cncf.io/). The versions are usually available in Kubernetes Kapsule and Kubernetes Kosmos within a few days after the release.
 
 <Message type="important">
-  Each minor Kubernetes version is supported by Scaleway Kubernetes for at least twelve months before being deprecated.
+  Scaleway is committed to provide updates on the products lifecycle as follows:
+
+  - Notification of feature deprecation will be announced in the [changelog](/changelog/).
+  - Information regarding a feature's End of Life (EOL) will be communicated through tickets and [scheduled maintenance notices](https://status.scaleway.com/).
```

Comment:
Can we specify "support tickets in the console" so it's clearer?

Reply:
```suggestion
  - Information regarding a feature's End of Life (EOL) will be communicated through support tickets in the console and [scheduled maintenance notices](https://status.scaleway.com/).
```